engine/src/main/java/org/hibernate/validator/internal/util/annotation/AnnotationDescriptor.java

```diff
@@ -191,20 +198,16 @@ private int buildHashCode() {
 
 			int nameHashCode = member.getKey().hashCode();
 
-			int valueHashCode =
-					!value.getClass().isArray() ? value.hashCode() :
-							value.getClass() == boolean[].class ? Arrays.hashCode( (boolean[]) value ) :
-									value.getClass() == byte[].class ? Arrays.hashCode( (byte[]) value ) :
-											value.getClass() == char[].class ? Arrays.hashCode( (char[]) value ) :
-													value.getClass() == double[].class ? Arrays.hashCode( (double[]) value ) :
-															value.getClass() == float[].class ? Arrays.hashCode( (float[]) value ) :
-																	value.getClass() == int[].class ? Arrays.hashCode( (int[]) value ) :
-																			value.getClass() == long[].class ? Arrays.hashCode(
-																					(long[]) value
-																			) :
-																					value.getClass() == short[].class ? Arrays
-																							.hashCode( (short[]) value ) :
-																							Arrays.hashCode( (Object[]) value );
+			int valueHashCode = !value.getClass().isArray() ? value.hashCode()
+					: value.getClass() == boolean[].class ? Arrays.hashCode( (boolean[]) value )
+					: value.getClass() == byte[].class ? Arrays.hashCode( (byte[]) value )
+					: value.getClass() == char[].class ? Arrays.hashCode( (char[]) value )
+					: value.getClass() == double[].class ? Arrays.hashCode( (double[]) value )
+					: value.getClass() == float[].class ? Arrays.hashCode( (float[]) value )
+					: value.getClass() == int[].class ? Arrays.hashCode( (int[]) value )
+					: value.getClass() == long[].class ? Arrays.hashCode( (long[]) value )
+					: value.getClass() == short[].class ? Arrays.hashCode( (short[]) value )
+					: Arrays.hashCode( (Object[]) value );
```

Comment:
Thanks for cleaning this up!

Reply:
I just wasn't able to do other stuff in that file looking at that formatting :)

Reply:
Ah ah, yeah, I just close my eyes when I work on this file but I agree it's a bit harder to code this way :).